docs/source/library_reference.rst

```diff
@@ -336,6 +334,10 @@ Logging
     :show-inheritance:
     :no-members:
 
+.. autodata:: strip_ansi
+
+.. automodule:: cocotb.ANSI
```

Comment:
This doesn't create anything that could act as a heading:
https://cocotb--4900.org.readthedocs.build/en/4900/library_reference.html#cocotb.logging.strip_ansi

Not sure right now how to best fix this...

Reply:
Maybe a `:synopsis:` already helps?

Reply:
Is the synopsis not the same thing as the module docstring?

Reply:
It's taken for the short descriptions in the index (https://cocotb--4900.org.readthedocs.build/en/4900/py-modindex.html) where nobody ever looks; but here I was just hoping it would also trigger a change in the module docs rendering. But what you have also works.

Reply:
Actually... if we used "proper" enums or something like that in the ANSI module, this problem would solve itself and the user could see the values we have defined.